cmds/record.c

```diff
@@ -1474,9 +1474,9 @@ static void print_child_usage(struct rusage *ru)
 
 #define UFTRACE_MSG  "Cannot trace '%s': No such executable file.\n"
 
-#define MCOUNT_MSG  "Can't find '%s' symbol in the '%s'.\n"		\
+#define MCOUNT_MSG  "Cannot find '%s' symbol in the '%s'.\n"		\
 "\tIt seems not to be compiled with -pg or -finstrument-functions flag\n" 	\
```

Comment:
I think it's better to add a period ('.') since it became the end of the statement.

Reply:
I fixed them.